### PR TITLE
Update maintainers. Bad encoding.

### PR DESCRIPTION
**Pull request by 130s**
Addressing suggestion at https://github.com/ros-planning/moveit_msgs/commit/c5c4c6181e3e9a24ac1f8020e1ad3dbd05e13b00#commitcomment-18705574

### CODE REVIEW

**Comment by v4hn** [comment]
+1
Do we have the maintainer changes in I&K already or does this need to be ported?

**Comment by 130s** [comment]
I think so, and I will do.

File: package.xml [hunk 1]
```diff
@@ -5,7 +5,11 @@
   <author email="isucan@google.com">Ioan Sucan</author>
   <author email="sachinc@sri.com">Sachin Chitta</author>
 
-  <maintainer email="isucan@google.com">Ioan Sucan</maintainer>
+  <maintainer email="dave@dav.ee">Dave Coleman</maintainer>maintainer>
+  <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>maintainer>
+  <maintainer email="me@v4hn.de">Michael Goerner</maintainer>maintainer>
+  <maintainer email="rhaschke@techfak.uni-bielefeld.de">Robert Haschke</maintainer>maintainer>
+  <maintainer email="130s@2000.jukuin.keio.ac.jp">Isaac I. Y. Saito</maintainer>maintainer>
```
Comment (davetcoleman) [inline package.xml]: The maintainer tag is malformed!

Comment (v4hn) [inline package.xml]: On Mon, Aug 22, 2016 at 12:27:23AM -0700, Dave Coleman wrote:
> The maintainer tag is malformed!

Crap. Fixed in #28. Please merge it there.

**Pull request merged**